crates/router/src/core/admin.rs

```diff
@@ -421,9 +409,7 @@ pub async fn create_payment_connector(
     let mca = store
         .insert_merchant_connector_account(merchant_connector_account)
         .await
-        .map_err(|error| {
-            error.to_duplicate_response(errors::ApiErrorResponse::DuplicateMerchantConnectorAccount)
-        })?;
+        .to_duplicate_response(errors::ApiErrorResponse::DuplicateMerchantConnectorAccount)?;
```

Comment:
Can you add a field called `connector_label` which will provide an information about what was the connector label that was created, because we construct this field.

Reply:
Sure. If this not urgent we could add it as good first issue.